CHARTER.md

```diff
@@ -0,0 +1,232 @@
+# JSON Schema Charter
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## 0: Guiding Principles
+The JSON Schema project is part of the OpenJS Foundation. The JSON Schema project strives to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how we wish to operate. As such, we define the following charter which includes aspects of the governance model to which we subscribe and by which we operate.
+
+## 1: Scope
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. We aim to enable these additional and emergent use cases.
+
+### 1.1: In-scope
+The scope of the JSON Schema project is split into two sections: primary and secondary concerns.
+Primary concerns are areas we wish to give focus to, while secondary concerns are more likely to require the desire of the community to drive the work.
+
+Primary Concerns
+- Publication of the JSON Schema standard
+  - Validation of JSON data
```

Comment:
Do we want to clarify here as "data that is compatible with the JSON document model"? Occasionally I hear "oh I can't use JSON Schema because this is YAML" which is showing a misunderstanding of what level the schemas are operating at.

Reply:
Maybe more simply, "JSON-compatible data"?

Reply:
I would actually argue no for this. There are quite a few data formats which are comaptible with the JSON document model. We can't concern ourselves with being able to validate YAML for example. People can convert the YAML to JSON, but I don't think we should be calling out YAML or any other format that converts to JSON. IMHO that would be too broad a concern.

Reply:
I think we actually operate on our own definition of a JSON data model. So we can't say "The JSON data model".

Open to further thoughts.

Reply:
From [Section 4.2.1](https://json-schema.org/draft/2020-12/json-schema-core.html#name-json-schema-documents):

> JSON Schema interprets documents according to a data model. A JSON value interpreted according to this data model is called an "instance".
> 
> An instance has one of six primitive types, and a range of possible values depending on the type:
> 
> null: A JSON "null" value
> boolean: A "true" or "false" value, from the JSON "true" or "false" value
> object: An unordered set of properties mapping a string to an instance, from the JSON "object" value
> array: An ordered list of instances, from the JSON "array" value
> number: An arbitrary-precision, base-10 decimal number value, from the JSON "number" value
> string: A string of Unicode code points, from the JSON "string" value

While we say "_a_ data model," we also lay out exactly the data model described by JSON.  (I suppose, technically, we use "boolean" whereas JSON defines `true` and `false` tokens.)

Reply:
The data model we define also includes equality, section 4.2.2.

Reply:
Resolved with https://github.com/json-schema-org/community/pull/325/commits/fbe8b54623e127e698454ad947253aa501e00414 after discussing on an OCWM: https://github.com/json-schema-org/community/issues/420